linux_os/guide/system/software/integrity/crypto/configure_openssl_crypto_policy/ansible/shared.yml

```diff
@@ -9,6 +9,14 @@
   {{%- set openssl_cnf_path="/etc/pki/tls/openssl.cnf" %}}
 {{% endif %}}
 
+{{% if product in ["fedora", "ol9", "rhel9"] %}}
```

Comment:
I think it would be good for the simplicity and prosperity of this rule to simplify the remediation without using `=` in all products. There is no technical impact on it.